src/asset.ts

```diff
@@ -230,4 +230,23 @@ export default class Asset extends AinftBase {
     const trailingUrl = `credit/${symbol}/withdraw/complete`;
     return this.sendRequest(HttpMethod.POST, trailingUrl, body);
   }
+  
```

Comment:
minor) Could you add description as jsDoc?

Reply:
I added description lockupUserBalance!
I will soon add a description of all the functions I added.